.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
Add entry `.idea` for files generated by intellij with IDE settings not being pushed to the repository.